content/millennium/r4/clinical/medications/immunization.md

```diff
@@ -15,6 +15,10 @@ Detailed administration records may be found in MedicationAdministration, while
 
 An immunization reaction may indicate an allergy or intolerance. If so, a separate AllergyIntolerance resource instance should be created as well.
 
+  * The following [HL7® FHIR® US Core Implementation Guide STU 4.0.0](https://hl7.org/fhir/us/core/STU4/) Profiles are supported by this resource:
+
+    * [US Core Immunization Profile](http://hl7.org/fhir/us/core/STU4/StructureDefinition-us-core-immunization.html)
+
 The following fields are returned if valued:
 
 * [Id](http://hl7.org/fhir/R4/resource-definitions.html#Resource.id){:target="_blank"}
```

Comment:
Need to add a line for Identifier

Reply:
I'm not sure what should I do here @cerbs

Reply:
Add an entry to this list following the existing patterns. It should link to https://hl7.org/fhir/R4/immunization-definitions.html#Immunization.identifier and be called something like "Immunization Record Identifier"

The new item should be second in the list, immediately after Id

Reply:
Solved in [be2f9fe](https://github.com/cerner/fhir.cerner.com/pull/884/commits/be2f9fecf4f90e67cb9f07d8c72f80afd2ace091)